bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/Clip2ThingHandler.java

```diff
@@ -0,0 +1,1128 @@
+                return;
+        }
+
+        if (putResource == null) {
+            logger.warn("{} -> handleCommand() command:{} not supported on channelUID:{}", resourceId, command,
+                    channelUID);
+            return;
+        }
+
+        putResourceId = Objects.nonNull(putResourceId) ? putResourceId : commandResourceIds.get(putResource.getType());
+        if (putResourceId == null) {
+            logger.warn("{} -> handleCommand() channelUID:{}, command:{}, putResourceType:{} => missing resource ID",
+                    resourceId, channelUID, command, putResource.getType());
+            return;
+        }
+
+        if (DYNAMIC_CHANNELS.contains(channelId)) {
+            if (Instant.now().isBefore(dynamicsExpireTime) && !dynamicsDuration.isZero()
+                    && !dynamicsDuration.isNegative()) {
+                if (ResourceType.SCENE == putResource.getType()) {
+                    putResource.setRecallDuration(dynamicsDuration);
+                } else {
+                    putResource.setDynamicsDuration(dynamicsDuration);
+                }
+            }
+        }
+
+        putResource.setId(putResourceId);
+        logger.debug("{} -> handleCommand() put resource {}", resourceId, putResource);
+
+        try {
+            getBridgeHandler().putResource(putResource);
+        } catch (ApiException | AssetNotLoadedException e) {
+            logger.warn("{} -> handleCommand() error {}", resourceId, e.getMessage(), e);
+        }
+    }
+
+    /**
+     * Handle a 'dynamics' command for the given channel ID for the given dynamics duration.
+     *
+     * @param channelId the ID of the target channel.
+     * @param command the new target state.
+     * @param duration the transition duration.
+     */
+    public synchronized void handleDynamicsCommand(String channelId, Command command, QuantityType<?> duration) {
+        if (DYNAMIC_CHANNELS.contains(channelId)) {
+            Channel dynamicsChannel = thing.getChannel(CHANNEL_2_DYNAMICS);
+            Channel targetChannel = thing.getChannel(channelId);
+            if (Objects.nonNull(dynamicsChannel) && Objects.nonNull(targetChannel)) {
+                logger.debug("{} - handleDynamicsCommand() channelId:{}, command:{}, duration:{}", resourceId,
+                        channelId, command, duration);
+                handleCommand(dynamicsChannel.getUID(), duration);
+                handleCommand(targetChannel.getUID(), command);
+                return;
+            }
+        }
+        logger.warn("{} - handleDynamicsCommand() failed for channelId:{}, command:{}, duration:{}", resourceId,
+                channelId, command, duration);
+    }
+
+    @Override
+    public void initialize() {
+        Clip2ThingConfig config = getConfigAs(Clip2ThingConfig.class);
+
+        String resourceId = config.resourceId;
+        if (Objects.isNull(resourceId) || resourceId.isEmpty()) {
```

Comment:
See comment in `Clip2ThingConfig` :
```suggestion
        if (resourceId.isBlank()) {
```

Reply:
Ok. Done.